package.json

```diff
@@ -103,6 +103,7 @@
     "redux-thunk": "^2.2.0",
     "rxjs": "^5.5.6",
     "tslint": "^5.9.1",
-    "utility-types": "^1.0.0"
+    "utility-types": "^1.0.0",
+    "websocket": "^1.0.25"
```

Comment:
we are not using `websocket` and `utility-types`. However we are missing the express package